test/services/network/test_network_service_comparator.py

```diff
@@ -923,3 +930,15 @@ def test_compare_loop(self):
             lambda _: Substation(mrid="s1"),
             lambda _: Substation(mrid="s2")
         )
+
+    def test_compare_feeder(self):
```

Comment:
todo: This test overrides the previous test with the same name and prevents it from running. The IDE should be warning you about this.